code/game/machinery/camera/presets.dm

```diff
@@ -28,49 +28,49 @@ var/global/list/engineering_networks = list(
 	network = list(NETWORK_CRESCENT)
 
 /obj/machinery/camera/network/fist_section
-	network = list(NETWORK_FIRST_SECTION)
+	network = list(NETWORK_AI, NETWORK_FIRST_SECTION)
```

Comment:
why was it necessary to insert NETWORK_AI everywhere here? 

It would be a lot better to give the AI a list of networks it can use, surely?